src/system/SystemPacketBuffer.cpp

```diff
@@ -755,18 +769,20 @@ PacketBufferHandle PacketBufferHandle::CloneData() const
         size_t originalDataSize       = original->MaxDataLength();
         uint16_t originalReservedSize = original->ReservedSize();
 
-        if (originalDataSize + originalReservedSize > PacketBuffer::kMaxSizeWithoutReserve)
+        uint32_t maxSize = PacketBuffer::kMaxAllocSize;
```

Comment:
do we need a separate variable? Should this be const?

To have minimal change we should probably not create a separate variable.